.circleci/config.yml

```diff
@@ -1345,6 +1345,41 @@ jobs:
           name: Verify Values Match
           command: |
             ./ops/scripts/ci-match-values-between-files.sh "<< parameters.file1_path >>" "<< parameters.pattern_file1 >>" "<< parameters.file2_path >>" "<< parameters.pattern_file2 >>"
+
+  test-proofs:
+    machine:
+      image: ubuntu-2204:2023.07.2
+    steps:
+      - checkout
+      - check-changed:
+          patterns: contracts-bedrock/test/kontrol,contracts-bedrock/src/L1/OptimismPortal\.sol
+      - run:
+          name: "Start Docker Container"
+          command: |
+            export KONTROL_VERSION=$(cat ../../.kontrolrc)
```

Comment:
Can we move this logic into a shell script so its portable and possible to run locally more easily? there is a `scripts` directory in `contracts-bedrock`

Reply:
Cc @F-WRunTime